app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -235,6 +234,7 @@ def create_response_info
             image_metrics: parse_image_metrics,
             address_line2_present: !pii_from_doc[:address2].blank?,
             classification_info: classification_info,
+            liveness_enabled: @liveness_checking_enabled,
```

Comment:
Slack thread about this [here](https://gsa-tts.slack.com/archives/C05HSH9RQ57/p1706629148029889).